app/src/main/java/org/oppia/android/app/options/AppLanguageFragment.kt

```diff
@@ -43,31 +58,29 @@ class AppLanguageFragment : InjectableFragment(), AppLanguageRadioButtonListener
     container: ViewGroup?,
     savedInstanceState: Bundle?
   ): View? {
-    val args =
-      checkNotNull(arguments) { "Expected arguments to be passed to AppLanguageFragment" }
-    val prefsKey = args.getStringFromBundle(APP_LANGUAGE_PREFERENCE_TITLE_ARGUMENT_KEY)
-    val prefsSummaryValue = if (savedInstanceState == null) {
-      args.getStringFromBundle(APP_LANGUAGE_PREFERENCE_SUMMARY_VALUE_ARGUMENT_KEY)
-    } else {
-      savedInstanceState.get(SELECTED_LANGUAGE_SAVED_KEY) as String
-    }
+
+    val oppiaLanguage =
+      checkNotNull(
+        savedInstanceState?.retrieveLanguageFromSavedState()
+          ?: arguments?.retrieveLanguageFromArguments()
+      ) { "Expected arguments to be passed to AudioLanguageFragment" }
```

Comment:
Nit: please update error to be consistent with this class's name.

Reply:
Thanks updated.